### PR TITLE
Sign up and Login pages 19 & 32 - Last Attempt

### PR DESCRIPTION
**Pull request by Mars-Cat2023**
## Describe your changes
This is the last version of my Sign Up and Login Page.
All the reported issues in Slack and GitHub.

(The only different thing is that the Sign Up is a little bit higher than the Don't have an account? 
@dave-cao Hope that you can fix this!)
By the way, I need to wait until Forgot Password Page is ready, then to add that link. (edited)

## Issue ticket number and link
19 & 32

## Screenshots / Video

image
屏幕截图 2024-07-10 064308

[Screen_recording_20240710_065308.webm](https://github.com/Firm-Collective/ai-journal/assets/153325945/78277484-6967-47ec-a4a2-2622c88d2c9b)

## Checklist
- [x] I have performed a self-review of my code
- [x] I have used `npx expo-doctor` to review any dependency issues.
- [x] I have pulled and merged any recent changes from the main branch (no merge conflicts)
- [x] I have tested thoroughly any edge cases that might break the application
- [x] I have deleted any console.log statements and useless comments

### CODE REVIEW

**Comment by Lyton505** [comment]
There's a back button slightly off on my screen[1]. Is the dark circle around Google a design?[2]

**Comment by Lyton505** [comment]
It's only an issue on Android. On iPhone it doesn't exist

**Comment by Mars-Cat2023** [comment]
To David and Lyton:
For Problem2, you can find that black circle on Figma. Because we all agree to strictly follow the format designed by our design team, I just make that one deliberately. 

The problem1 found by Lyton is very abnormal, and @dave-cao why he can still see that upper bar? In fact, I use and test on Android, it is not a problem. I don't know why. @dave-cao try your best to solve it. Otherwise,  I will be back next week to start from this.

**Comment by Mars-Cat2023** [comment]
@dave-cao If it is just a number or just slightly different from ideal cases, just put it there. Later, we can still use a new PR to change it if necessary. That's what we need to talk with designers later, because there is no one format suitable and ideal for all phones. The compatibility among different types of mobiles is a common problem when designing apps.

**Comment by Mars-Cat2023** [comment]
By the way, problem 1 would also appear in David's page too, because I just reuse his interface.
If I remember correct, that is caused by two engineers choose to use the 'Absolute' value (absolute position), which I don't like, especially not good when you consider different size of your screen.
We can collaborately try to find an alternative way about compatibility.

**Comment by dave-cao** [comment]
> There's a back button slightly off on my screen[1]. Is the dark circle around Google a design?[2] 

@Lyton505 Did you clone from Qilongs fork? Make sure that within `app/(auth)/_layout.tsx`, the headerShown for both the login and signup pages are false
image

**Comment by Lyton505** [comment]
Yes, I am on Qilong's fork, and yes the header is gone. There's now a red line and it's due to this [line](https://github.com/Firm-Collective/ai-journal/blame/b35fc8b8bf2c6da42d3d6c61a0980f5e8083de38/app/(auth)/login.tsx#L15). @dave-cao

**Comment by dave-cao** [comment]
> Yes, I am on Qilong's fork, and yes the header is gone. There's now a red line and it's due to this [line](https://github.com/Firm-Collective/ai-journal/blame/b35fc8b8bf2c6da42d3d6c61a0980f5e8083de38/app/(auth)/login.tsx#L15). @dave-cao

@Lyton505 Fixed it. That was weird.

**Comment by Mars-Cat2023** [comment]
> Yes, I am on Qilong's fork, and yes the header is gone. There's now a red line and it's due to this [line](https://github.com/Firm-Collective/ai-journal/blame/b35fc8b8bf2c6da42d3d6c61a0980f5e8083de38/app/(auth)/login.tsx#L15). @dave-cao

**Could you upload the picture of the red line?** 
You shouldn't see that, as long as your app is occupied the whole screen and well implemented. It is a deliberate mechanism to make sure that nowhere is unimplemented in our app. (To help figure out the bug during development.)

**Comment by Mars-Cat2023** [comment]
> > Yes, I am on Qilong's fork, and yes the header is gone. There's now a red line and it's due to this [line](https://github.com/Firm-Collective/ai-journal/blame/b35fc8b8bf2c6da42d3d6c61a0980f5e8083de38/app/(auth)/login.tsx#L15). @dave-cao
> 
> **Could you upload the picture of the red line?** You shouldn't see that, as long as your app is occupied the whole screen and well implemented. It is a deliberate mechanism to make sure that nowhere is unimplemented in our app. (To help figure out the bug during development.)

♡I haven't encounted this scenario before and cannot imagine that edge cases. Put your picture here.

**Comment by Mars-Cat2023** [comment]
@dave-cao About the size of apple, I just think that it is too small with a very very big F or seems not that fair. So I adjust to make them to balance between them.
Of course, you are right.

**Review by Lyton505 — APPROVED** [review]
Changes seem on point

File: app/(auth)/signup.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+const styles = StyleSheet.create({
+  Screen: {
+    flex: 1,
+    backgroundColor: '#FF0000',
+  },
+});
+
+export default Signupp;
```
[the remark this answers is not in the document]
Comment (dave-cao) [reply app/(auth)/signup.tsx]: Refresh the page, it should be updated.

**Review by dave-cao — APPROVED** [review]

**Pull request merged**